test/mustachio/foo.dart

```diff
@@ -0,0 +1,9 @@
+@Renderer(#renderFoo, Context<Foo>())
```

Comment:
Since this is checked in it probably belongs in the grinder's rebuild check?

Reply:
Done!